examples/mecci_kissinger.py

```
@@ -0,0 +1,234 @@
+
+    plt.tight_layout()
+    plt.show()
+    plt.close(fig)
+
+    # Now that the figure is closed, we can use findpeaks
+    fp = findpeaks(lookahead=5, interpolate=10, method='topology')
+    try:
+        if exothermic:
+            results = fp.fit(smoothed_hf)
+        else:
+            results = fp.fit(-smoothed_hf)
```

Ruff

Use ternary operator `results = fp.fit(smoothed_hf) if exothermic else fp.fit(-smoothed_hf)` instead of `if`-`else`-block

Replace `if`-`else`-block with `results = fp.fit(smoothed_hf) if exothermic else fp.fit(-smoothed_hf)`

(SIM108)